recipe/recipe.yaml

```diff
@@ -0,0 +1,86 @@
+context:
+  name: py-rattler
+  python_name: py_rattler
+  version: "0.9.0"
+  python_min: "3.9.*"
+
+recipe:
+  name: py-rattler
+  version: ${{ version }}
+
+source:
+  url: https://pypi.org/packages/source/${{ name[0] }}/${{ name }}/${{ python_name }}-${{ version }}.tar.gz
+  sha256: 488922a45ed3276732f9c413617b6ed6587eb0583af28e9c5ab48e946e8b1a25
+
+build:
+  number: 1
+
+outputs:
+  - package:
+      name: py-rattler
+    build:
+      # only build for the minimal python version
+      skip: not match(python, python_min)
```

Comment:
This needs a check for `is_abi3` too for future proofing for freethreading builds.

Reply:
Done! :)